src/components/VotingPage/VotingPage.js

```diff
@@ -0,0 +1,16 @@
+import styled from 'styled-components/macro'
+
+export default function VotingPage() {
+  return (
+    <VotingPageLayout>
+      Reading is to the mind what exercising is to the body. ✨
+    </VotingPageLayout>
+  )
+}
+const VotingPageLayout = styled.div`
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  font-size: 200%;
```

Comment:
I think it's better to stick to fixed values like 24px instead of the percentage.